### DISCUSSION TITLE
Drag-and-arrange 2FA tokens in preferred order, instead alphabetically

### DISCUSSION THREAD

**Discussion by BeemoKincaid** [ente-io/ente#3427] [Enhancements]
Hope there is an option that I can arrange my 2FA tokens in preferred order, not just alphabetically by default. I love putting my Facebook account 2FA token on top spot, actually.

**Comment by ua741** [comment]
You can pin the code if you want it on top.
This seems duplicate of https://github.com/ente-io/ente/discussions/497
